### ISSUE TITLE
Missing label on recaptcha form field (accessibility)

### ISSUE THREAD

**Issue by sneelz** [DethAriel/ng-recaptcha#202]
### Summary

I'm submitting a: 
- [x] bug report
- [ ] feature request
- [ ] question / support request 
- [ ] other

### Description

When you run a webpage with the recaptcha on it through accessibility software like WAVE, it calls out a missing form field on the recaptcha. It seems to be on a textarea field that is not actually visible to the user until you turn the page styles off. I'm not sure if this is specific to ng-recaptcha or recaptcha overall.

### Demo

Instructions:
1. Go to any site using ng-recaptcha (e.g. https://dethariel.github.io/ng-recaptcha/basic)
2. Run WAVE on the page: https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh

Example screenshot of ng-recaptcha WAVE result with styles:

![image](https://user-images.githubusercontent.com/11659471/100632022-58ebe700-32fa-11eb-840c-e7d18755ef9a.png)

Example screenshot of ng-recaptcha WAVE result without styles:

![image](https://user-images.githubusercontent.com/11659471/100632009-54273300-32fa-11eb-9776-b19e2e8eb209.png)

- [x] I did my best to come up with a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve)

Lib versions:
* ng-recaptcha: 5.0.0
* Angular: 9.1.12
* Typescript (`tsc --version`): 3.8.3

**Comment by DethAriel** [comment]
Hi @sneelz, and thanks for submitting this.

This `textarea` field is added by reCAPTCHA core itself, which the lib has no control over. I can only assume why it's there in the first place (probably for easier HTML form values manipulation). It isn't supposed to be edited by users -- it's value is controlled programmatically IIUC.

I see you've already created [another issue](https://github.com/google/recaptcha/issues/421) in the [google/recaptcha project](https://github.com/google/recaptcha/), so I'm going to close this one.
